selfdrive/athena/athenad.py

```diff
@@ -472,6 +474,32 @@ def log_handler(end_event):
       cloudlog.exception("athena.log_handler.exception")
 
 
+def stat_handler(end_event):
+  while not end_event.is_set():
+    last_scan = 0
+    try:
+      curr_scan = sec_since_boot()
+      if curr_scan - last_scan > 10:
+        for stat_entry in os.listdir(STATS_DIR):
+          stat_path = os.path.join(STATS_DIR, stat_entry)
+          with open(stat_path) as f:
+            stats = json.load(f)
+          jsonrpc = {
+            "method": "storeStats",
+            "params": {
+              **stats
+            },
+            "jsonrpc": "2.0",
+            "id": stat_entry
+          }
+          low_priority_send_queue.put_nowait(json.dumps(jsonrpc))
+          os.remove(stat_path)
+        last_scan = curr_scan
```

Comment:
Should you set this in case of an exception? Otherwise it might get into a tight loop if it keeps failing.